app/controllers/JourneyAnswersController.scala

```diff
@@ -55,21 +54,68 @@ class JourneyAnswersController @Inject()(propertyService: PropertyService,
     }
   }
 
+  def saveExpenses(taxYear: TaxYear, businessId: BusinessId, nino: Nino, incomeSourceId: IncomeSourceId): Action[AnyContent] =
```

Comment:
As discussed, we don't require two separate APIs for saving and updating

Reply:
Built this based on @modivilioglu 's implementation for Income. We van refactor this when we build the UI.

Reply:
lets clean up part of this PR,  you check @modivilioglu other PRs

Reply:
I'd suggest we go forward, and I'll apply the change you request in my ticket @arunkumar461 , because I'm already gonna rebase mine on top of this and then I'll implement your request on it. I'd suggest we get this ticket out of the way asap. Thanks